contrib/runners/mistral_v2/query/mistral_v2.py

```diff
@@ -87,7 +94,14 @@ def _get_workflow_result(self, exec_id):
         :type exec_id: ``str``
         :rtype: (``str``, ``dict``)
         """
-        execution = self._client.executions.get(exec_id)
+        try:
+            execution = self._client.executions.get(exec_id)
+        except mistralclient_base.APIException as mistral_exc:
+            if 'not found' in mistral_exc.message:
+                raise exceptions.ReferenceNotFoundError(mistral_exc.message)
+            raise mistral_exc
+        except Exception:
```

Comment:
This is redundant since we are not doing any handling of the exception here so it can be removed.